ToDoTime/ClientApp/src/pages/tasksListDetails/TasksListDetails.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "@emotion/styled";
+import { MainContainerTemplate } from "shared/components/MainContainerTemplate";
+import { useLoaderData } from "react-router-dom";
+import { OneEightyRingWithBg } from "react-svg-spinners";
+import { APP_GOLD_COLOR } from "shared/constants/appStyles";
+import { useFetchTasksForList } from "./useFetchTasksForList";
+import { NoTasks } from "pages/layoutComponents/NoTasks";
+import { ListOfTasks } from "pages/layoutComponents/ListOfTasks";
+import { CreateNewTaskInline } from "pages/layoutComponents/CreateNewTaskInline";
+import { useManageListOfTasks } from "./useManageListOfTasks";
+
+export const TasksListDetails: React.FC = () => {
+  const listId = useLoaderData() as number;
+  const tasksList = useFetchTasksForList(listId);
+  const { listOfTasks, toggleTaskDone, onCreateTask } = useManageListOfTasks(tasksList?.tasks);
+
+  if (!tasksList) {
+    return (
+      <MainContainerTemplate title="Loading...">
+        <TasksWrapper>
+          <OneEightyRingWithBg color={APP_GOLD_COLOR} width={150} height={150} />
+        </TasksWrapper>
+      </MainContainerTemplate>
+    );
+  }
+
+  return (
```

Comment:
Should also have a way to edit the description.

Reply:
Not only the description, but the title as well.

Reply:
Edit for description added, although still need to add support for click to finish editing, instead of hitting enter

Reply:
> Edit for description added, although still need to add support for click to finish editing, instead of hitting enter

Added icons to confirm edit in description, still missing edit for title

Reply:
> Not only the description, but the title as well.

Added the editable input for the title

Reply:
Code looks good